cmd/ghalistener/metrics/metrics.go

```diff
@@ -224,7 +224,7 @@ func (b *baseLabels) jobLabels(jobBase *actions.JobMessageBase) prometheus.Label
 	return prometheus.Labels{
 		labelKeyEnterprise:     b.enterprise,
 		labelKeyOrganization:   b.organization,
```

Comment:
Hello! 

Should we do the same here? 
`b.organization` will be empty for runners assigned to an enterprise. 
```suggestion
		labelKeyOrganization:   jobBase.OwnerName,
```

Reply:
Thanks!